cloudformation/add_cloudflare_ips_to_sgs.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python3
 
-# TODO: move this into Ansible or at least use Boto
-
-import json
-import subprocess
+import boto3
 import requests
+from botocore.exceptions import ClientError
+
+SECURITY_GROUP_ID = "sg-0e07161e54ca34212"
```

Comment:
We could probably just query the security group list for a tag like `CloudFlareIngress` and iterate over the results, which would come in handy if e.g. we wanted to have staging match production in this aspect, too.

Reply:
Oh I like that idea. I'm going to ticket that.